src/js/steps/checkLanguageChange.js

```diff
@@ -10,7 +10,7 @@ module.exports = function ($monkeyContainer, options, pickerLocales) {
   // We need to check whether we have to show the language overlay, and we do
   // this by seeing whether the current language is different to the book's
   // language when it was initialized, whether we're switching between locales
-  // which have the character picker (currently en-GB and en-US) and therefore
+  // which have the character picker (currently en-GB, en-US, fr) and therefore
```

Comment:
I feel this is something that shouldn't really be in a comment. Won't it go out of date the moment someone who isn't you adds a locale?